railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -69,9 +69,9 @@ def credentials_generator
 
         def missing_credentials_message
           if Rails.application.credentials.key.nil?
-            "Missing master key to decrypt credentials. See bin/rails credentials:help"
+            "Missing master key to decrypt credentials. See rails credentials:help"
           else
-            "No credentials have been added yet. Use bin/rails credentials:edit to change that."
+            "No credentials have been added yet. Use rails credentials:edit to change that."
```

Comment:
Messages like this should really be using backticks to set the command off from the surrounding sentence (or a "words words words: command until end of line" form). It was not-great before, but I think the obvious command-ness of `bin/rails` at least made the start clear... `rails` slides right into the sentence.

This and encrypted_command below seem to be the only ones -- might as well fix them.